backend/errors/domain.error.js

```diff
@@ -0,0 +1,13 @@
+class DomainError extends Error {
+    constructor(message) {
+      super(message);
+     // Ensure the name of this error is the same as the class name
+      this.name = this.constructor.name;
+     // This clips the constructor invocation from the stack trace.
+     // It's not absolutely essential, but it does make the stack trace a little nicer.
+     //  @see Node.js reference (bottom)
```

Comment:
Looks like you copied too much of the comment from the example. We don't have a reference to node.js docs here. Please remove this line.